Added elastic-agent compatibility (#4549)
elastic/integrations: packages/cloud_security_posture/docs/README.md

1. Amazon EKS on AWS Outposts

This integration does not currently support the security posture assessment of the managed kubernetes services below:

1. Google GKE
2. Azure AKS 
3. Red Hat Openshift 
4. Amazon EKS with AWS Fargate nodes

> This integration currently supported for **elastic agent** version 8.5 and above.

Reviewer: ```suggestion
The integration supports **elastic agent** version 8.5 and above.
```
